File: dao/src/main/java/com/alta_v2/game/dao/data/map/MapEntity.java
package com.alta_v2.game.dao.data.map;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;
import lombok.Getter;

@Getter
@DatabaseTable(tableName = "maps")
public class MapEntity {

    @DatabaseField(id = true)
    private int id;

    @DatabaseField
    private String folder;

    @DatabaseField
    private String name;

    @DatabaseField(columnName = "display_name")
    private String displayName;

    @DatabaseField(columnName = "internal_description")
    private String internalDescription;

    public String getMapPath() {
        return this.folder + this.name;
    }
}

File: dao/src/main/java/com/alta_v2/game/dao/data/map/MapEntity.kt
package com.alta_v2.game.dao.data.map

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

@DatabaseTable(tableName = "maps")
class MapEntity {

    @DatabaseField(id = true)
    var id: Int = 0

    @DatabaseField
    lateinit var folder: String

    @DatabaseField
    lateinit var name: String

    @DatabaseField(columnName = "display_name")
    lateinit var displayName: String

    @DatabaseField(columnName = "internal_description")
    lateinit var internalDescription: String

    val mapPath: String
        get() = folder + name
}